src/labels_by_user_input.py

```diff
@@ -35,3 +38,8 @@ def labels_by_user_input(event_data, pull):
         if f"/un{LABEL_APPROVE}".lower() in body:
             remove_label(pull=pull, label=LABEL_APPROVE)
             set_commit_status_pending_no_approve(commit=last_commit)
+
+    if semver_labels_exist_in_pr(pull, body):
+        set_commit_status_success_semver(commit=last_commit)
```

Comment:
I expected to see here `add_remove_semver_labels()` along with  `set_commit_status_success_semver()`
this keep the flow above and make it more undertood.

Reply:
I would much rather find an appropriate name for `semver_labels_exist_in_pr()` to retain the minimal implementation in `labels_by_user_input.py`. I don't think we should move the logic from `utils.py` to this file.
Would you be open to this?

Reply:
I think we should keep the same logic as for other labels 
https://github.com/myakove/pr-workflows/pull/46

Reply:
For me the logic that should be here is:
```
if <something>: # something can be function that do the logic for us
    add_label()
    set_commit_status()
```

Reply:
done